packages/firebase_ml/example/pubspec.yaml

```diff
@@ -18,7 +18,7 @@ dependencies:
   cupertino_icons: ^0.1.3
 
   image_picker: ^0.6.7
-  tflite: 1.0.6
+  tflite: 1.1.1
```

Comment:
Change required for the code to compile. For some reason 1.0.6 is totally broken on IOS